.github/workflows/checks.yaml

```diff
@@ -24,6 +24,22 @@ jobs:
           node ./scripts/build-libs/command.mjs
           yarn run test
 
+      - name: Hash Current Branch Name
+        id: hash_curret_branch_name
+        run: |
+          hashed_name=$(echo -n "${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" | base64)
+          echo "hashed=${hashed_name}">> $GITHUB_OUTPUT
+
+      - name: Upload Build Meta
```

Comment:
I think one assumption here is this step will be run after `Test` step which is running `node ./scripts/build-libs/command.mjs` to build all the packages. That would be great to add a comment on top this step to make the assumption clear.